pkg/cmd/types.go

```diff
@@ -9,6 +9,19 @@ var (
 	ScanOptions = []string{"openscap"}
 )
 
+type MultiStringVar struct {
+	Values []string
+}
+
+func (sv *MultiStringVar) Set(s string) error {
```

Comment:
In our case Yes. I am implementing an interface here for the ```flag``` library: https://golang.org/src/flag/flag.go?s=6274:6334#L230